### PR TITLE
Add new consumption api to partner guide

### PR DESCRIPTION
**Pull request by bgrenon**
While adding the 2 new consumption APIs to the partner guide, also pulled the querying metrics content out of the single page into a standalone page for querying via API. This will simplify repurposing the metrics-api page for more general usage if/when it no longer belongs sequestered in the Partner guide.

Preview:
- https://neon-next-git-bgrenon-add-new-consumption-a-ca0828-neondatabase.vercel.app/docs/guides/partner-billing
- https://neon-next-git-bgrenon-add-new-consumption-a-ca0828-neondatabase.vercel.app/docs/guides/metrics-api

### CODE REVIEW

**Comment by danieltprice** [comment]
We mention the pro plan, which no longer exists:
"free tier users can consume versus pro or enterprise users"
https://neon-next-git-bgrenon-add-new-consumption-a-ca0828-neondatabase.vercel.app/docs/guides/partner-billing

**Comment by danieltprice** [comment]
"using the quote object" <- should be quota
"https://neon-next-git-bgrenon-add-new-consumption-a-ca0828-neondatabase.vercel.app/docs/guides/partner-billing#corresponding-quotas"

File: content/docs/guides/metrics-api.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+| [Granular project-level metrics](https://api-docs.neon.tech/reference/getconsumptionhistoryperproject)   | Provides detailed metrics for each project in an account at a specified granularity level (e.g., hourly, daily, monthly) | Scale plan only   |
+| [Billing period project-level metrics](https://api-docs.neon.tech/reference/listprojectsconsumption)     | Offers consumption metrics for each project in an account for the current billing period                                 | All plans         |
+| [Single project metrics](https://api-docs.neon.tech/reference/getproject)                                | Retrieves detailed metrics and quota information for a specific project                                                  | All plans         |
+
+## Get account-level aggregated metrics
+
+Using the [Get account consumption metrics API](https://api-docs.neon.tech/reference/getconsumptionhistoryperaccount)), you can find total usage across all projects in your organization. This provides a comprehensive view of consumption metrics accumulated for the billing period.
```
Comment (danieltprice) [inline content/docs/guides/metrics-api.md:22]:
```suggestion
Using the [Get account consumption metrics API](https://api-docs.neon.tech/reference/getconsumptionhistoryperaccount), you can find total usage across all projects in your organization. This provides a comprehensive view of consumption metrics accumulated for the billing period.
```

File: content/docs/guides/metrics-api.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+
+You can set `from` and `to` query parameters, plus a level of granularity to define a time range that can span across multiple billing periods.
+
+- `from` — Sets the start date and time of the time period for which you are seeking metrics.
+- `to` — Sets the end date and time for the interval for which you desire metrics.
+- `granularity` — Sets the level of granularity for the metrics, such as `hourly`, `daily`, or `monthly`.
+
+The response is organized by periods and consumption data within the specified time range.
+
+[Details on setting a date range](#details-on-setting-a-date-range)
```
Comment (danieltprice) [inline content/docs/guides/metrics-api.md:50]: Does this link need some context, like "For information about ..., see x."

**Review by danieltprice — APPROVED** [review]
LGTM! A couple of minor comments.

File: content/docs/guides/metrics-api.md [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
```
Comment (bgrenon) [inline content/docs/guides/metrics-api.md:50]:
```suggestion
See [Details on setting a date range](#details-on-setting-a-date-range) for more info.
```

File: content/docs/guides/metrics-api.md [hunk 4]
```diff
@@ -0,0 +1,408 @@
...
+
+### Pagination
+
+To control pagination (number of results per response), you can include these query parameters:
+
+- `limit` — sets the number of project objects to be included in the response.
+- `cursor` — by default, the response uses the project `id` from the last project in the list as the `cursor` value (included in the `pagination` object at the end of the response). Generally, it is up to the application to collect and use this cursor value when setting up the next request.
+
+[Details on pagination](#details-on-pagination)
```
Comment (bgrenon) [inline content/docs/guides/metrics-api.md:131]:
```suggestion
See [Details on pagination](#details-on-pagination) for more info.
```

File: content/docs/guides/metrics-api.md (hunk 2 above)
Comment (bgrenon) [reply content/docs/guides/metrics-api.md:50]: Yes, changed.

File: content/docs/guides/metrics-api.md [hunk 5]
```diff
@@ -0,0 +1,408 @@
...
+    // ... More projects
+  ]
+}
+```
+
+</details>
+
+The response is organized by periods and consumption data within the specified time range.
+
+[Details on setting a date range](#details-on-setting-a-date-range)
```
Comment (bgrenon) [inline content/docs/guides/metrics-api.md:122]:
```suggestion
See [Details on setting a date range](#details-on-setting-a-date-range) for more info.
```

**Pull request merged**